.github/workflows/integration-tests.yml

```diff
@@ -0,0 +1,48 @@
+# SPDX-FileCopyrightText: Copyright 2024 Move Agency Group B.V., Zwolle, The Netherlands
+# SPDX-License-Identifier: CC0-1.0
+
+name: Integration Tests
```

Comment:
We already have "integration tests" in https://github.com/move-backend/ApnsPHP/blob/master/.github/workflows/php-tests.yml#L50. If we want these tests separate, I'd say move the other ones here too.